src/components/DelimitedList/_index.scss

```diff
@@ -5,8 +5,13 @@
 ////
 
 @import '../Component/mixins';
-@import 'mixins';
 
 @include component($name: delimited-list) {
-  @include delimited-list;
+  display: flex;
```

Comment:
Same question as the `src/components/ComboButton/_mixins.scss` file. I saw a note in your PR about redundant mixin files but wondering what qualifies as a redundant mixin files for future work. Thanks

Reply:
See https://github.com/carbon-design-system/ibm-security/pull/114#discussion_r330932907